catalog/transformer/catalog-transformer-generic-api/src/main/java/ddf.catalog.transformer.generic.xml.api/Gml3ToWkt.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.generic.xml.api;
```

Comment:
I would also suggest using org.codice.ddf as the base package and replace the use of generic.  So the folder structure should be something like `org/codice/ddf/catalog/transformer/xml/streaming/` with no api and `org/codice/ddf/catalog/transformer/xml/streaming/impl` in the impl project.